src/event.h

```diff
@@ -59,7 +59,7 @@ typedef struct {
 	size_t pid_str_len;
 
 	pthread_t tid;
-	char tid_str[30 + 1];
+	char tid_str[56 + 1];
```

Comment:
56 is a magic number which comes from onwhere. Why 56? Why not 42 as a global answer?